apps/speed.c

```diff
@@ -124,10 +124,19 @@
 #define MAX_ECDH_SIZE   256
 #define MISALIGN        64
 
+struct sec {
```

Comment:
I think make this a typedef, "typedef struct sec_st { .... } SEC" or similar.
And use time_t for the counters.

Reply:
okay for the typedef.
problem with time_t is portability e.g. wrt conversion format specifier. The number of seconds was treated as int before i saw no reason to change this.

Reply:
okay that makes sense.  this looks good to me.